sound/sequences/00_sound_player.s

```diff
@@ -1,4 +1,5 @@
 #include "seq_macros.inc"
+#include "config.h"
```

Comment:
can you just include config_audio?

Reply:
Only including audio would probably save you from recompiling sound every single time you toggle a debug option, which might be valuable time saved with enough streamed audio.

Reply:
nah the only reason we included everything in config.h was mostly because of most files including config.h lol. if you include that then changing any config option will recompile audio which would slow things down even more. instead including just config_audio means that only changing config audio would recompile this file

Reply:
I wrote this before seeing arctics comment

Reply:
gotcha will change it then